crates/weblsp/src/main.rs

```diff
@@ -49,34 +49,25 @@ fn main_loop(
     init_params: serde_json::Value,
     mut css_language_service: csslsrs::service::LanguageService,
 ) -> Result<ExitCode, Box<dyn Error + Sync + Send>> {
-    let mut awaiting_exit = false;
     let _init_params: InitializeParams = serde_json::from_value(init_params).unwrap();
 
     for msg in &connection.receiver {
-        // TODO: Handle trace levels and notifications instead of just printing them to stderr.
-        eprintln!("new msg: {:?}", msg);
```

Comment:
This log is not necessary, you can set `weblsp.trace.server` to either `messages` or `verbose` for the client to output what messages you received